### ISSUE TITLE
gcp longevity test cn io operation timeout reached

### ISSUE THREAD

**Issue by huangjw806** [risingwavelabs/risingwave#17452]
```
================================================================================
longevity-test Result
================================================================================
Result               FAIL                
Pipeline Message     v1.9.2-jianwei-rt gcp test
Namespace            usrlngvty-20240625-133617
TestBed              medium-3cn-all-affinity
RW Version           v1.9.2-jianwei-rt   
Test Start time      2024-06-25 13:44:29 
Test End time        2024-06-25 13:45:39 
Test Queries         nexmark_q0,nexmark_q1,nexmark_q2,nexmark_q3,nexmark_q4,nexmark_q5,nexmark_q6_group_top1,nexmark_q7,nexmark_q8,nexmark_q9,nexmark_q10,nexmark_q12,nexmark_q14,nexmark_q15,nexmark_q16,nexmark_q17,nexmark_q18,nexmark_q19,nexmark_q20,nexmark_q21,nexmark_q22,nexmark_q101,nexmark_q102,nexmark_q103,nexmark_q104,nexmark_q105
Grafana Metric       https://grafana.test.risingwave-cloud.xyz/d/EpkBw5W4k/risingwave-dev-dashboard?orgId=1&var-datasource=Prometheus:%20test-gcp-usce1-gke-a&var-namespace=usrlngvty-20240625-133617&from=1719323069000&to=1719323139000
Grafana Logs         https://grafana.test.risingwave-cloud.xyz/d/liz0yRCZz1/log-search-dashboard?orgId=1&var-data_source=Logging:%20test-gcp-usce1-gke-a&var-namespace=usrlngvty-20240625-133617&from=1719323069000&to=1719323139000
Memory Dumps         https://s3.console.aws.amazon.com/s3/buckets/test-useast1-mgmt-bucket-archiver?region=us-east-1&bucketType=general&prefix=k8s/usrlngvty-20240625-133617/&showversions=false
Buildkite Job        https://buildkite.com/risingwave-test/longevity-test/builds/1530
```

**Comment by hzxa21** [comment]
@wcy-fdu Can you help take a look to see whether the issue is resolved or do we need some action items (e.g. tune some default configs for gcs)?

**Comment by wcy-fdu** [comment]
I think this issue is resolved, as the [longevity test on 7.1 passed](https://buildkite.com/risingwave-test/longevity-test/builds/1549) after running for 12h(This failed job was run on 6.25).
